dashboard/lib/widgets/sign_in_button.dart

```diff
@@ -48,14 +51,18 @@ class SignInButton extends StatelessWidget {
                   break;
               }
             },
-            child: WebImage(
-              // TODO(chillers): Switch to use avatar widget provided by google_sign_in plugin
-              imageUrl: authService.user?.photoUrl,
-              placeholder: Padding(
-                padding: const EdgeInsets.only(right: 10.0, top: 20.0),
-                child: Text(authService.user!.email),
-              ),
-            ),
+            iconSize: Scaffold.of(context).appBarMaxHeight,
+            icon: Builder(builder: (BuildContext context) {
+              if (!kIsWeb && Platform.environment.containsKey('FLUTTER_TEST')) {
```

Comment:
The following condition was necessary to mimic the [previous functionality that WebImage](https://github.com/flutter/cocoon/blob/6ec0130b56ef2256427509dea61235d14ad11fc0/dashboard/lib/widgets/web_image.dart#L26) provided and pass the existing unit tests. See a7bef171279ca90f2df418b7e6ad47cc28c21eb5

I originally intended to wrap the WebImage with a CircleAvatar. This would have reduced the code-complexity of this PR. At the same time, I wanted to respect the intended implementation in the original TODO by using GoogleUserCircleAvatar. Either approach achieves the same functionality.